Assets/MixedRealityToolkit.Tests/TestUtilities.cs

```diff
@@ -135,8 +135,18 @@ public static void InitializeMixedRealityToolkitAndCreateScenes(bool useDefaultP
             InitializeMixedRealityToolkit(useDefaultProfile);
         }
 
+        public static void InitializeCamera()
+        {
+            if (Camera.main == null)
+            {
+                new GameObject("Main Camera", typeof(Camera), typeof(AudioListener)) { tag = "MainCamera" }.GetComponent<Camera>();
```

Comment:
For my own education, why did you need the .GetComponent<Camera>() at the end there?

Reply:
Ooh...looks like more was copied over from the `CameraCache` script than necessary. Doesn't do anything in this case and could be removed